### DISCUSSION TITLE
当前网页的title能否显示在标题栏?

### DISCUSSION THREAD

**Discussion by xung** [tw93/Pake#308] [Q&A]
hi,我之前是用nativefier打包的,它打包的应用可以在标题栏显示网页的title(当网页的title变化时,打包的应用的标题栏也会跟着变)
例如:我用nativefier打包的[code-server](https://github.com/coder/code-server):
![image](https://user-images.githubusercontent.com/2482978/218033496-c6022766-469f-403c-a98d-ae5860f79d7a.png)

这样的好处时,如果我打开多个应用实例(例如在code-server中,我会一个项目开一个应用实例), 我再选择不同的项目时, 看标题就可以区分出来了:
![image](https://user-images.githubusercontent.com/2482978/218034723-75e17efe-0def-4ea3-a145-eb911be3fbf3.png)

而用pake打包的应用, 在标题栏是空的, 无法快速区分项目:
![image](https://user-images.githubusercontent.com/2482978/218034455-587a2057-754b-4717-9cae-058d2dc187e1.png)

可否把当前网页的title能否显示在标题栏?

**Comment by tw93** [comment]
可以的 下次迭代时候我们加上
